boilerplate/openshift/golang-osd-operator-acceptance-test/update

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Global vars
+REPO_ROOT=$(git rev-parse --show-toplevel)
```

Comment:
You'll get this var for free when this script is invoked from the main `update` driver.

Reply:
Ah good to know! Removed.